packages/sources/svelte/src/components/structure/VtmnCard/VtmnCard.svelte

```diff
@@ -24,12 +24,20 @@
    */
   export let contentOpaque = false;
 
+  /**
+   * @type {1|2|3|4|5|6} Heading level of card title
+   * @default 2
+   */
+  export let headingLevel = 2;
+
   let className = undefined;
   /**
    * @type {string} Custom classes to apply to the component.
    */
   export { className as class };
 
+  const Heading = `h${headingLevel}`;
```

Comment:
We need to make a reactive var here like 

```suggestion
  $: titleElement = headingLevel > 0 && headingLevel <= 6 && `h${headingLevel}` || 'span';
```